ReactAndroid/src/main/java/com/facebook/react/common/ViewMethodsUtil.java

```diff
@@ -0,0 +1,19 @@
+package com.facebook.react.common;
+
+import android.view.View;
+
+/**
+ * Various utility methods for {@link View}.
+ */
+public class ViewMethodsUtil {
```

Comment:
Android has a `ViewUtil` class.  Would it be confusing if we define one as well?